om4-header-footer-code.php

```diff
@@ -13,27 +13,27 @@
 */
 
 /*
+	Copyright 2012-2016 OM4 (email: plugins@om4.com.au    web: http://om4.com.au/)
```

Comment:
Perhaps it's time for a copyright date year bump:

```suggestion
	Copyright 2012-2024 OM4 (email: plugins@om4.com.au    web: http://om4.com.au/)
```

?

Reply:
Yes, that makes sense. Thanks! It escaped my attention.